plugins/CoreHome/Controller.php

```diff
@@ -38,11 +44,22 @@ class Controller extends \Piwik\Plugin\Controller
      */
     private $translator;
 
+    /**
+     * @var FeatureFlagManager
+     */
+    private $featureFlagManager;
+
     public function __construct(Translator $translator)
     {
         $this->translator = $translator;
 
         parent::__construct();
+
+        $this->featureFlagManager = new FeatureFlagManager(
+            [new ConfigFeatureFlagStorage(Config::getInstance())],
+            [Example::class],
```

Comment:
Should we get the list of classes here from the DI?

Reply:
Ideally all of it from the DI, I'm playing it safe though because the DI config for FeatureFlags won't be available in until the plugin has been activated, so doing it this way just ensures nothing breaks hard.

As this is a bit of code that will be removed soon, I'm not too fussed.

Reply:
And the line can actually be removed now after the refactoring.

Reply:
Fixed this 👍